sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/IncrementalExecution.scala

```diff
@@ -259,6 +259,24 @@ class IncrementalExecution(
     }
   }
 
+  /**
+   * This rule populates the column family schemas for the TransformWithStateExec
+   * operator to ship them from the driver, where the schema and serializer objects
+   * are created, to the executor.
+   */
+  object StateStoreColumnFamilySchemasRule extends SparkPlanPartialRule {
+    override val rule: PartialFunction[SparkPlan, SparkPlan] = {
+      case statefulOp: StatefulOperator =>
+        statefulOp match {
+          case op: TransformWithStateExec =>
```

Comment:
nit: I assume you're doing this two step matching, because the avro serde will be added to other operators too in follow ups?

Reply:
Yeah, it will be.